### DISCUSSION TITLE
How to debug the PassthroughCluster traffic

### DISCUSSION THREAD

**Discussion by garyyang85** [kiali/kiali#4178] [General]
Just enabled service mesh in my project. Have created vs and dr for all services, but seems there is still some traffic route to "PassthroughCluster". I am wondering if there is a way to debug how the "PassthroughCluster" generated and what traffic got there?

"Requests going to PassthroughCluster (or BlackHoleCluster) are requests that did not get routed to a defined
service or service entry, and instead end up at one of these built-in Istio request handlers."

**Comment by israel-hdez** [comment]
Hi, @garyyang85 

Kiali is limited to what telemetry can offer. Unfortunately, for the PassthroughCluster traffic, there isn't a lot of information about what external host was called.

As an idea, you may try to check the sidecar logs of the pods that generated that traffic and see if they show any hostname/ip that you haven't configured yet. But it's just an idea and I'm not totally sure that the sidecar logs will contain that information.

**Reply by jshaughn** [reply]
You can change your external traffic setting from ALLOW_ANY to REGISTRY_ONLY and that should block all Passthrough and send it to BlackHole, but the requests will fail.  It may help to identify things.
